client/src/components/ButtonSubmit/ButtonSubmit.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./ButtonSubmit.css";
+
+// Function based React Component
+const ButtonSubmit = props => {
+  // Default Class to apply to Component
+  let classList = "ButtonSubmit";
+  let types = [
+    "followed",
+    "favorited",
+    "next",
+    "finish",
+    "login",
+    "joinTribe",
+    "register",
+    "connect",
+    "message",
+    "join",
+    "book",
+    "follow",
+    "post"
+  ];
```

Comment:
There aren't really different types of buttons - just some different styles. Like the "followed" and "favorited" buttons are the same, they just have different text.